app/views/handlers/view_submitted_response_pdf.py

```diff
@@ -54,7 +54,8 @@ def __init__(
     @property
     def filename(self) -> str:
         """The name to use for the PDF file"""
-        return f"{self._metadata['schema_name']}.pdf"
+        if self._questionnaire_store.submitted_at is not None:
+            return f"{self._metadata['schema_name']}-{self._questionnaire_store.submitted_at.date()}.pdf"
```

Comment:
In here it should be probably something like 
```python
return f"{self._schema.json['title']}-{self._questionnaire_store.submitted_at.date()}.pdf
```

Reply:
Spaces replaced by hyphens so something like:
```python
f"{self._schema.json['title']}-{self._questionnaire_store.submitted_at.date().strftime('%Y-%m-%d')}.pdf".replace(" ", "-")
```

Reply:
👍 added all this except `strftime` wasn't needed `date` already outputs the correct format

Reply:
I think we should use `strftime()` as it converts date explicitly, `date()` doesn't outputs a string but a date object, implicit type conversion is something we would like to avoid.

Reply:
Yeah but `.replace()` and `.lower()` will both return a string anyway. So adding `strftime()` wont really make a difference

Reply:
As discussed, `.replace()` and `.lower()` are methods of string type so by the time you execute both of them the var is already converted implicitly to string.

Reply:
I think either is fine but either way our test should ensure that we get the format that it expects, so that if the underlying `__str__` format of date changed, our test should break. (The format shouldn't change in the stdlib). But then again, explicit is better than implicit. If it were to change, I think we should change it to `date.isoformat()` instead of `strftime()` as ISO 8601 is a spec, so the format is explicit.

Reply:
1. I think it would be better for the call to `lower()` and `replace()` to be done just on the title instead of the entire string. 
2. Most schemas also use ` - ` separator in the title, which would result a title `MBS - Construction and Allied Trades` to be transformed into `mbs---construction-and-allied-trades`, therefore `--` needs to be replaced with `-` so that we end up with `mbs-construction-and-allied-trades`